src/Adapters/__mockData__/3DScenesConfiguration.json

```diff
@@ -199,7 +201,11 @@
                         "url": "https://cardboardresources.blob.core.windows.net/cardboard-mock-files/refinery_scene_textured_1m_2.glb",
                         "extensionProperties": {}
                     }
-                ]
+                ],
+                "pollingConfiguration": {
+                    "pollingStrategy": "Limited",
```

Comment:
are those ()e.g. "Limited" reserved strings? If so, can we define a enum type in the 3d config schema and use it here?

Reply:
ahh I just noticed that, in fact there is a type for it, please ignore my comment